templates/cloud/foundation-cloud/index.html

```diff
@@ -0,0 +1,194 @@
+        <h4 class="u-no-margin--bottom"><a href="/cloud/contact-us?product=foundation-cloud"  onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - get started' : undefined });">Talk to us&nbsp;&rsaquo;</a></h4>
+      </div>
+      <div class="col-4 p-card u-align--center">
+        <a href="https://insights.ubuntu.com/wp-content/uploads/9ee0/Datasheet_Foundation-cloud-build_02.17.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - get started' : undefined });"><img src="{{ ASSET_SERVER_URL }}3e9e430a-icon-documentation.svg" alt="" style="max-height: 58px; margin-bottom: 12px" /></a>
+        <h4 class="u-no-margin--bottom"><a href="https://insights.ubuntu.com/wp-content/uploads/9ee0/Datasheet_Foundation-cloud-build_02.17.pdf" class="p-link--external" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - get started' : undefined });">Read the datasheet</a></h4>
+      </div>
+      <div class="col-4 p-card u-align--center">
+        <a href="{{ ASSET_SERVER_URL }}580fb240-FBC+Hardware+Requirements_AW.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Hardware requirements', 'eventLabel' : 'Hardware requirements - get started' : undefined });"><img src="{{ ASSET_SERVER_URL }}4d7dbd0c-icon-settings.svg" alt="" style="max-height: 58px; margin-bottom: 12px" /></a>
+        <h4 class="u-no-margin--bottom"><a href="{{ ASSET_SERVER_URL }}580fb240-FBC+Hardware+Requirements_AW.pdf" class="p-link--external" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Hardware requirements', 'eventLabel' : 'Hardware requirements - get started' : undefined });">Hardware requirements</a></h4>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--x-light is-bordered">
+    <div class="row">
+      <h2>Features</h2>
+      <ul class="p-list is-split">
+        <li class="p-list__item is-ticked">Two-day onsite requirements gathering workshop</li>
+        <li class="p-list__item is-ticked">Tools for ongoing operations management are included</li>
+        <li class="p-list__item is-ticked">KVM and LXD hypervisors</li>
+        <li class="p-list__item is-ticked">Ceph-based storage</li>
+        <li class="p-list__item is-ticked">Full HA implementation of all control services in converged or discrete architecture</li>
+        <li class="p-list__item is-ticked">Automation and monitoring packages such as Juju, Prometheus, or Nagios and more are included as standard</li>
```

Comment:
Automation and monitoring packages such as Juju, Prometheus, Nagios, and more are included as standard.